source/SymAnimate.h

```diff
@@ -171,13 +184,29 @@ class SymAnimate : public UI::Animate {
 
   }
 
+
+  /**
+   * Input: The string representing the button identification. 
+   * 
+   * Output: None
+   * 
+   * Purpose: To add style to the buttons displayed. 
+   */
   void setButtonStyle(std::string but_id){
     auto but = buttons.Button(but_id);
     but.SetCSS("background-color", "#D3D3D3");
     but.SetCSS("border-radius", "4px");
     but.SetCSS("margin-left", "5px");
   }
 
+
+  /**
+   * Input: The address of the canvas being used. 
```

Comment:
It's being passed by reference, which means that the input is the object, and C++ handles only passing the address, but I think the input shouldn't mention the address since the user won't use it that way